packages/web-client/app/components/card-drop-page/card/index.hbs

```diff
@@ -0,0 +1,34 @@
+<Boxel::CardContainer class='card-drop-page-card' ...attributes>
+  <div class='card-drop-page-card__section'>
+    {{yield}}
+
+    <h3 class='card-drop-page-card__heading'>
+      {{yield to='heading'}}
+    </h3>
+
+    <p class='card-drop-page-card__explanation'>
+      {{yield to='explanation'}}
+    </p>
+
+    {{#if this.ua.isMobile}}
```

Comment:
`isMobile` doesn't really reflect the criteria we use, which is that the platform must support installation of the CardWallet app. There are a lot more OSes besides Android and iOS that do not.

If we want a convenience method to refer to `isAndroid` and `isIOS`, a more appropriately named one is `canInstallWallet`.